src/components/sortingFunctions.js

```diff
@@ -10,13 +10,13 @@ export const getShoppingItemBackgroundStyles = (
   );
 
   if (daysBetweenCurrentAndPreviousPurchase >= daysLeftForNextPurchase * 2) {
-    return 'grey';
+    return '#9CA3AF';
   } else if (daysLeftForNextPurchase <= 7) {
-    return 'green';
+    return '#EF4444';
```

Comment:
I'm not so sure if this color is a good choice for this case :thinking:. It looks like a lighter red, very similar to the other red. I would either use some shade of green, or use a monochromatic palette (ex. red) for all of the buttons, and the darker the shade, the quicker you would have to buy the item.
I'd like to hear some more opinions on this :slightly_smiling_face:

Reply:
Great idea with the monochromatic!! 👍🏽  This was anyway just to show something, we didn't decide on the final colors yet.

Reply:
I changed it to mono. No worries on how this fits to the app green, which color we are going in the end will be chosen during  next week 🙂